resources/views/layout/main.blade.php

```diff
@@ -156,10 +156,10 @@
     </div>
     <footer class="bg-dark text-center text-light pb-4">
         <div class="d-flex flex-column justify-content-center align-items-center">
-            <p class="mt-5"><a class="text-decoration-underline text-light"
-                    href="https://projects.hanisirfan.xyz/seajell" target="_blank">SeaJell</a> {{ $appVersion }}</p>
+            <p class="mt-5"><a class="text-decoration-underline text-light" href="https://www.seajell.xyz/"
+                    target="_blank">SeaJell</a> {{ $appVersion }}</p>
             <p>Hak Cipta &copy; <a href="http://hanisirfan.xyz" class="text-light">Muhammad Hanis Irfan bin Mohd
-                    Zaid</a> 2021</p>
+                    Zaid</a> {{ date('Y') }}</p>
```

Comment:
I'm sorry but as for my readings, copyright notice date shouldn't be updated dynamically.
> Which Year Should You Use in Your Copyright Notice?
>
> The general rule is that the year to include in a copyright notice is the year of first publication of the work. First publication is when the work is made available to the public without restriction. This includes selling a book or leaving free copies of it in a public place or posting content on your blog. Publication does not include, for example, circulating copies internally to co-workers or sending copies of a book to a publisher.

> For compilations or derivative works which incorporate previously published content, the year of first publication of the compilation or derivative work is sufficient.

So leaving the date to the first year of publication is sufficient. And also, can you change the notice from my name to `The SeaJell Contributors`? 

This is because you're now joining as one of the project contributors. I think it's better to include copyright notice this way.